src/style.css

```diff
@@ -0,0 +1,40 @@
+* {
```

Comment:
do not need to reset margin in all elements, will be better to add only to elements which needed